aesd-char-driver/aesd-circular-buffer.c

```diff
@@ -100,3 +100,31 @@ void aesd_circular_buffer_init(struct aesd_circular_buffer *buffer)
 {
     memset(buffer,0,sizeof(struct aesd_circular_buffer));
 }
+
+size_t aesd_circular_buffer_size(struct aesd_circular_buffer *buffer) 
+{
+    size_t size = 0;
+    uint8_t i;
+    for (i = 0; i < AESDCHAR_MAX_WRITE_OPERATIONS_SUPPORTED; i++) 
```

Comment:
Fine for this simple example but it would be more efficient to start from outptr and increment to inptr rather than always count every element.